src/client/components/equalizer/canvasequalizer/canvasequalizer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Graphicequaliser=(props)=>{
+    return(
+        <div>
+            <canvas id="graphicequalizer" width={props.width} height={props.height}> 
+            </canvas> 
+            <br />
+            <input type="range" min="100" max="1000" step="10" 
+                    defaultValue={props.width} name="rangecanvas" id="range" onChange={props.onchange}/>
+        </div>);
+    };
+
```

Comment:
I think that the correct way to write with spaces and use CamelCase. For example `const GraphicEqualizer = (props) => {...`
And also should get rid of `return` if it is the only method in the class. For example 
` const GraphicEqualizer = (props) => (
       <div> .... </div>
)`